### DISCUSSION TITLE
Class \"Facade\\Ignition\\LogRecorder\\LogMessage\" not found

### DISCUSSION THREAD

**Discussion by pitylee** [facade/ignition#445] [Q&A]
### Discussed in https://github.com/facade/ignition/discussions/443

<div type='discussions-op-text'>

<sup>Originally posted by **pitylee** December 20, 2021</sup>
Hello,

Anyobody met this?

I have the basic configuration which comes with Laravel, and the logging config is the default one.

By simply using the Laravel Log::info() the following will happen every time:

```
{
    "message": "Class \"Facade\\Ignition\\LogRecorder\\LogMessage\" not found",
    "exception": "Error",
    "file": "/var/www/html/vendor/facade/ignition/src/LogRecorder/LogRecorder.php",
    "line": 39,
    "trace": [
        {
            "file": "/var/www/html/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php",
            "line": 404,
            "function": "record",
            "class": "Facade\\Ignition\\LogRecorder\\LogRecorder",
            "type": "->"
        },
        {
            "file": "/var/www/html/vendor/laravel/framework/src/Illuminate/Events/Dispatcher.php",
            "line": 249,
            "function": "Illuminate\\Events\\{closure}",
            "class": "Illuminate\\Events\\Dispatcher",
            "type": "->"
        },
....
```</div>

**Comment by indigoram89** [comment]
+1

**Comment by tm1000** [comment]
@pitylee I've suddenly got the same issue. No idea how to solve it

After looking into this for a while it was pretty clear to me that this library (ignition) does not have any coding bugs. Yet for some reason composer's autoload could never find this unless I included it directly.

I tried blowing away composer.lock and the vendor file to no avail.

The only thing that worked was a composer self-upgrade (from 2.2.2 to 2.2.3) then blowing away composer.lock and the vendor file but I see nothing in the change log about this in composer..

Anyways after I did that I wasn't able to replicate it. 

My suggestion to others experiencing this is to do the following:

1. run `composer self-upgrade`
2. delete vendor and composer.lock
3. run `composer install && composer upgrade`
4. ???
5. Profit

**Reply by indigoram89** [reply]
We get this error when we enable OpCache in Forge.

**Reply by Majkie** [reply]
Happened to me recently, no idea what caused it, but in my case php restart solved it.

**Comment by sblomberg** [comment]
We were getting this error in our local development environments using Laravel 8 and checking this package's readme I found:

> Using Laravel 8 or above?
> 
> If you're on Laravel 8 or above, you can switch to [spatie/laravel-ignition](https://github.com/spatie/laravel-ignition), which is a drop-in replacement. Replace facade/ignition with "spatie/laravel-ignition": "^1.0" in your application's composer.json file.

Removing this plugin via `composer remove facade/ignition` resolved this error message.

**Comment by abdurrahmanok** [comment]
Hi

**Reply by abdurrahmanok** [reply]
Hi
